services/userService.js

```diff
@@ -0,0 +1,45 @@
+const bycrpt = require("bcrypt");
+
+const userDao = require("../models/userDao");
+const checkValidation = require("../utills/validation-check");
+
+const signup = async (email, password, name, birth, phoneNumber, address) => {
+  try {
+    if (await userDao.checkRegisterdEmail(email)) {
+      const err = new Error("ALREADY_EXCIST_EMAIL");
+      throw err;
+    }
+    if (await userDao.checkRegisterdPhoneNumber(phoneNumber)) {
+      const err = new Error("ALREADY_EXCIST_PHONE_NUMBER");
+      throw err;
+    }
+
+    await checkValidation.checkValidationEmail(email);
+    await checkValidation.checkValidationPassword(password);
+    await checkValidation.checkValidationBirth(birth);
+
+    const costFactor = 12;
+    const makeHash = async (password, costFactor) => {
+      return await bycrpt.hash(password, costFactor);
+    };
+    const hashedPassword = await makeHash(password, costFactor);
```

Comment:
```suggestion
    const hashedPassword = await bycrpt.hash(password, 12);
```
함수내 함수라면 전역적으로 사용될 수 없기 때문에 모듈화하기 어렵고, 현재와 같은 간단한 로직이라면 함수를 만들어서 사용하는 것이 아닌 예시와 같이 곧바로 작성할 수 있게 해주세요